### DISCUSSION TITLE
cannot import name 'vuetify3' from 'trame.widgets' Falling back to a static output.   warnings.warn(

### DISCUSSION THREAD

**Discussion by theganeshpandey** [pyvista/pyvista#5436] [Q&A] [answered]
I am not sure if it is best to post on `trame` discussion forum. However, I am using pyvista with trame backend on jupyter. Hence posting this topic here. I often receive warnings like "cannot import name ..." after installing the latest version of trame (eg. version 2.5.0) however it works just fine with version 2.3.2.

**Comment by jourdain** [comment, marked as answer]
You should do `pip install pyvista[jupyter]` when using PyVista with [jupyter](https://github.com/pyvista/pyvista/blob/main/pyproject.toml#L49-L58) (and trame). 

Technically, PyVista when leveraging trame is using `trame`, `trame-vuetify` and `trame-vtk` as dependencies. Then for Jupyter, it is also good to have `trame-jupyter-extension` for using the existing websocket connection of Jupyter (especially for cloud deployment). 

HTH
